app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -790,66 +816,106 @@
         <item>Nada</item>
         <item>@string/configAction_setAlarm</item>
         <item>@string/configAction_setTimeZone</item>
-        <item>Mostrar próxima nota</item>
-        <item>Mostrar nota anterior</item>
+        <item>Exibir próxima nota</item>
+        <item>Exibir nota anterior</item>
+    </string-array>
+    <string-array name="clockTapActions_values" translatable="false">  <!-- config values (one-to-one with clockTapActions_display) -->
+        <item>NOTHING</item>
+        <item>ALARM</item>
+        <item>TIMEZONE</item>
+        <item>NEXT_NOTE</item>
+        <item>PREV_NOTE</item>
     </string-array>
 
     <!-- App Setting: Note UI "on tap" action -->
     <string-array name="noteTapActions_display">  <!-- display values (one-to-one with noteTapActions_values) -->
         <item>Nada</item>
         <item>Definir alarme para a nota</item>
-        <item>Mostrar próxima nota</item>
-        <item>Mostrar nota anterior</item>
+        <item>Exibir próxima nota</item>
+        <item>Exibir nota anterior</item>
+    </string-array>
+    <string-array name="noteTapActions_values" translatable="false">   <!-- config values (one-to-one with noteTapActions_display) -->
+        <item>NOTHING</item>
+        <item>ALARM</item>
+        <item>NEXT_NOTE</item>
+        <item>PREV_NOTE</item>
     </string-array>
 
     <!-- App Setting: time zone sort modes -->
-    <string name="timezone_sort_byOffset">by offset</string>  <!-- TODO -->
-    <string name="timezone_sort_byId">by id</string>  <!-- TODO -->
-    <string name="timezone_sort_byName">by name</string>  <!-- TODO -->
+    <string name="timezone_sort_byOffset">by offset</string>
+    <string name="timezone_sort_byId">by id</string>
+    <string name="timezone_sort_byName">by name</string>
 
     <string-array name="timezoneSort_display">  <!-- display values (one-to-one with timezoneSort_values) -->
         <item>@string/timezone_sort_byOffset</item>
         <item>@string/timezone_sort_byId</item>
         <item>@string/timezone_sort_byName</item>
     </string-array>
+    <string-array name="timezoneSort_values" translatable="false">   <!-- config values (one-to-one with timezoneSort_display) -->
+        <item>SORT_BY_OFFSET</item>
+        <item>SORT_BY_ID</item>
+        <item>SORT_BY_DISPLAYNAME</item>
+    </string-array>
 
-    <string name="timezone_sort_contextAction">Sort zones</string>           <!-- action bar title -->  <!-- TODO -->
-    <string name="timezone_sort_msg">Time zones sorted (%1$s).</string>      <!-- toast msg -->  <!-- TODO -->
+    <string name="timezone_sort_contextAction">Sort zones</string>           <!-- action bar title -->
+    <string name="timezone_sort_msg">Time zones sorted (%1$s).</string>      <!-- toast msg -->
 
     <!-- App Setting: GPS fix time limit -->
-    <string name="configLabel_getFix_maxElapse">Tempo limite do GPS</string>    <!-- TODO -->
```

Comment:
Is this a reversion? Same for `configLabel_getFix_maxElapse`, `configLabel_getFix_maxElapse_summary`, `configLabel_getFix_maxAge`, etc.